test/Microsoft.AspNetCore.TestHost.Tests/TestClientTests.cs

```diff
@@ -239,6 +239,54 @@ public void Dispose()
             clientSocket.Dispose();
         }
 
+        [ConditionalFact]
+        [FrameworkSkipCondition(RuntimeFrameworks.Mono, SkipReason = "Hangs randomly (issue #507)")]
```

Comment:
We no longer plan to test on Mono, we can remove all instance of this skip condition.